docs/content/getting-started.md

```diff
@@ -181,10 +183,10 @@ this:
 
 ## Modules Calling Modules
 
-This is all well and good, but we don't want to have to write things the same
-*every time* right? Well, good news: that's what modules are for! Now that
-you've written a module, you can require it from any other module to add it to
-your tree. Create a new module, let's call it `helloYou.hcl`:
+Let's look at how we can take advantage of reusability. Good news: that's what
+modules are for! Now that you've written a module, you can require it from any
+other module to add it to your tree. Create a new module, let's call it
```

Comment:
I see what you are saying. I was trying to be explicit b/c I think there could be confusion with how this relates to the previous example and that this new module is called `helloWorld.hcl` within a file named `helloYou.hcl`. I changed it up.